var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -75,47 +83,28 @@ def install(self, spec, prefix):
             # completely disabled. See
             # [SuiteSparse/SuiteSparse_config/SuiteSparse_config.mk] for more.
             'CUDA=no',
-            'CUDA_PATH={0}'.format(
```

Comment:
I prefer format because that's what Python 3 docs prefer, but both are valid and I won't reject a PR that uses the old syntax. I just think it's ugly